### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by cyyynthia**
Note that oauth callbacks are:
 - /oauth/discord
 - /oauth/spotify
 - /oauth/github

else stuff *should* work lol

### CODE REVIEW

**Comment by ohlookitsderpy** [comment]
yes pls now

**Comment by Bowser65** [comment]
aeth why are u playing fortnoot instead of reviewing my pr :crii:

**Comment by ohlookitsderpy** [comment]
absolutely disgusting behaviour aeth, im disappointed now

**Comment by aetheryx** [comment]
image

**Comment by Bowser65** [comment]
hmmmmmm

**Comment by aetheryx** [comment]
image

File: src/routes/dashboard/process.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+      name: req.body.name,
+      developer: req.body.developer,
+      manifest: null
+    });
+
+    // Create GH repo
+    await post(`https://api.github.com/orgs/${req.config.githubOrg}/repos`)
+      .set('Authorization', `token ${req.session.tokens.github.access_token}`)
+      .send({ name: req.body.name })
+      .execute();
```
Comment (aetheryx) [inline src/routes/dashboard/process.js]: you shouldn't need these .execute()s, just await post().set().send() will work

File: src/views/linking.ejs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      font-family: 'Uni Sans', sans-serif;
+    }
+  </style>
+</head>
+<body>
+<p style='display: none;'>You can close that window</p>
+<script>
+  fetch('http://localhost:6462', {
+    headers: {
+      'X-Powercord-Token': '<%= jwt %>'
```
Comment (aetheryx) [inline src/views/linking.ejs]: ideally i'd want this to be a POST with a body instead, transfering core payload over headers is less than ideal

**Comment by Bowser65** [comment]
Requested changes done :)

**Pull request merged**